### PR TITLE
Feat/ezc 201 homepage mobile

### PR DESCRIPTION
**Pull request by HoXuanTy**
Call api challenge home page

### CODE REVIEW

File: src/hooks/useChallange.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
```
Comment (ThaiHoang-newbie) [inline src/hooks/useChallange.ts]: Sai tên file: useChallenge.ts not useChallange.ts

File: package-lock.json
Comment (ThaiHoang-newbie) [inline package-lock.json:1]: remove this file

File: yarn.lock
Comment (ThaiHoang-newbie) [inline yarn.lock:1]: remove this file, pls

Comment (HoXuanTy) [reply yarn.lock:1]: finished

**Pull request merged**